compiler/rustc_feature/src/active.rs

```diff
@@ -576,6 +576,12 @@ declare_features! (
     /// Allows using and casting function pointers in a `const fn`.
     (active, const_fn_fn_ptr_basics, "1.48.0", Some(57563), None),
 
+    /// Allows trait bounds in `const fn`.
+    (active, const_fn_trait_bound, "1.53.0", Some(57563), None),
```

Comment:
I found no good tracking issue for this. https://github.com/rust-lang/rust/issues/57563 lists the following two for other aspects of the relevant RFC: https://github.com/rust-lang/rust/issues/67794, https://github.com/rust-lang/rust/issues/67792

Reply:
This is OK.